### PR TITLE
Increases Redis-Server somaxconn default TCP backlog settings to 511 instead of 128

### PR DESCRIPTION
**Pull request by DanielSpring**
Update docker-compose.yml

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -9,6 +9,9 @@ services:
   tanner_redis:
     build: ./redis
     container_name: tanner_redis
+    privileged: true
```
Comment (glaslos) [inline docker/docker-compose.yml]: is this required?

Comment (DanielSpring) [reply docker/docker-compose.yml]: privileged: true does not seem to be required.

Comment (glaslos) [reply docker/docker-compose.yml]: then please remove 👍

**Comment by glaslos** [comment]
Closing this now. Please submit a more isolated PR next time. You are changing multiple things at the same time some of which should not in this PR (Dockerfile, config changes...). I'm happy to help with a new PR

**Pull request closed without merging**